src/base/bittorrent/session.cpp

```diff
@@ -296,6 +296,16 @@ namespace
 
         return {};
     }
+
+    QString cachedGuid()
```

Comment:
Is `Session::networkInterface()` storing the same data? If yes, then you can reuse it.

Reply:
I think it stores the interface name and not the GUID.

Reply:
But they point to the same network device, right? I would prefer they use related names.
For example, `InterfaceGUID` for your case.